FAQ.md

```diff
@@ -2,11 +2,151 @@
+
+### Why do I need a VPS? Can't I just run BTCPay on my home computer?
+
+Just like you can host a website locally, you can do it with BTCPay. Please be aware that your PC would have to meet the minimal requirements and also run 24/7 if you don't want interruptions. While local hosting is good for testing, for production it's really not a viable solution.
+
+### Can the cost of running BTCPay on Microsoft Azure be reduced?
+
+Yes. Currently, the one-click deployment requires Azure Virtual Machine, but if you’re a programmer or understand VM’s you can deploy one on a different VM. Furthermore, the changes in the Bitcoin Core 0.16.0 (BIP159) make it possible to run the full node in the pruned mode to save more disk space.
+
+Also, once the full node is synchronized to the network, you can downgrade the machine on Azure to take a less expensive one.
+
+Please check out [How to penny-pinch your Azure deployment](AzurePennyPinching.md).
+
+## What are the minimal requirements to run BTCPay?
+
+If you would like to run Bitcoin and Lightning Network nodes, the minimal requirements are :
+- 2GB Ram
+- 250 GB of storage (for the initial sync) can be pruned to 80GB.
+- Docker
+
+## How to install BTCPay Server?
+
+There are several videos and article online which you can check out:
+
+- [How to install BTCPay on Azure - guide](https://github.com/btcpayserver/btcpayserver-doc/blob/master/AzureDeployment.md)
+- [How to install BTCPay on Azure - video](https://www.youtube.com/watch?v=Bxs95BdEMHY)
+- [How to install BTCPay on LunaNode VPS - article](https://medium.com/@BtcpayServer/hosting-btcpayserver-on-lunanode-bf9ef5fff75b)
+- [How to install BTCPay on LunaNode VPS - video](https://youtu.be/PZXwacAai9E)
+- [See all deployments](https://github.com/btcpayserver/btcpayserver-doc#deployment)
+
+## Why can't I just give my Bitcoin address to a buyer?
+
+Re-using the address for receiving payments is a privacy issue. Providing a different address manually to each customer is not an optimal solution. Imagine having to send an e-mail that wants to pay you with cryptocurrency.
+
+BTCPay solves the address re-use and automates the checkout process. The software is a complete, automated invoicing system. It integrates into your checkout process and customers can pay you in Bitcoin or altcoins in few clicks.
+
+After the payment, the software notifies your store that the order has been paid/completed. Depending on the e-commerce software you're using, it can also change the order status. You worry about getting items shipped, leave the invoicing and payment processing to BTCPay.
+
+## Does BTCPay need my private key?
+
+Absolutely not. The fact that BTCPay Server never needs access to your master private key is a huge security advantage. Even if your server gets hacked, your funds from the on-chain transactions are always safe. Securing your on-chain funds, comes down to [securing your wallet](https://btcinformation.org/en/secure-your-wallet).
```

Comment:
Exception: If you have a lightning node, BTCPay technically have access to the keys of LN.